index.html

```diff
@@ -15,18 +19,44 @@
 <!-- Load scatter-gl.js -->
 <script src="https://cdn.jsdelivr.net/npm/scatter-gl@0.0.1/lib/scatter-gl.min.js"></script>
 
+<!-- FONT
+–––––––––––––––––––––––––––––––––––––––––––––––––– -->
+<link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
+
+<!-- CSS
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+<link href="css/normalize.css" rel="stylesheet"/>
+<link href="css/skeleton.css" rel="stylesheet">
+</head>
+
```

Comment:
Wouldn't it be better to indent those so that they are clearly _inside_ `<head> </head>`?
Same goes for the content _inside_ `<body></body>`.
It's not a problem per se, I just think it would make the code more readable.
Speaking about code readability, shall we use a formatter like Prettier and perhaps, a linter as well like ESLint?

Reply:
Have addressed in https://github.com/misterpeddy/hands-down/pull/18